src/devices/MemoryLCD/AgBytes.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.MemoryLcd
+{
+    /// <summary>
+    /// Gate Line Address Setup
+    /// </summary>
+    internal static class AgBytes
+    {
+        public static byte[] Table { get; } = new byte[]
+        {
+            0b00000000,
```

Comment:
NIT: You don't have to fix this if you don't want to, but it is usually easier to read these guys when using an underscore as separators, for example: `0b_0000_0100`